extensions/roc-plugin-style-sass/src/index.js

```diff
@@ -32,11 +32,16 @@ export const roc = {
         extension: 'roc-plugin-style-css',
         hook: 'add-style',
         description: 'Adds Sass support to Webpack.',
-        action: ({ context: { config: { settings } } }) => () => () => ({
-            extensions: ['sass', 'scss'],
-            loaders: `${require.resolve('sass-loader')}?${settings.build.sass.useBourbon ?
-                `includePaths[]=${BOURBON_PATH}` : ''
-            }`,
-        }),
+        action: ({ context: { config: { settings } } }) => () => () => {
+            const params = [
+                settings.build.sass.useBourbon ? `includePaths[]=${BOURBON_PATH}` : '',
+                settings.build.style.sourceMap ? 'sourceMap' : '',
+            ].filter(v => v !== '');
+
+            return {
+                extensions: ['sass', 'scss'],
+                loaders: `${require.resolve('sass-loader')}${params.length > 0 ? '?' : ''}${params.join('&')}`,
```

Comment:
I think we can add `?` always, just to make the code a little easier to reason about since an empty query string is perfectly fine.

I would suggest moving `join` to the definition of the params, it will make it a little easier in my opinon to read the code.